app/controllers/work_packages_controller.rb

```diff
@@ -128,14 +128,20 @@ def authorize_on_work_package
   end
 
   def protect_from_unauthorized_export
-    if (supported_list_formats + %w[atom]).include?(params[:format]) &&
-       !User.current.allowed_to?(:export_work_packages, @project, global: @project.nil?)
-
+    if (supported_list_formats + %w[atom]).include?(params[:format]) && !user_allowed_to_export?
```

Comment:
Since you are introducing a method for this here, the name could also be inverted to something like `export_unauthorized?` as that would read better.

Reply:
Personally, I see this 100% the other way. `if not_allowed_to_something?` reads completely out of place to me. `if !allowed_to_something` is clearer. But if you feel strongly about it, I can change it